### PR TITLE
Add globbing to codestyle_exclude

### PR DESCRIPTION
**Pull request by TLATER**
This "works", but it doesn't quite satisfy test cases. Since I'm not
sure what this test case wants to assert, I'm a little stuck here.

### CODE REVIEW

**Comment by henry0312** [comment]
Hello, @TLATER,
I'm sorry for late response.
I don't have much time to review, but I will check #42 and #43 by this weekend.

File: pytest_codestyle.py [hunk 1]
```diff
@@ -20,14 +20,15 @@ def pytest_addoption(parser):
...
+    parser.addini('codestyle_exclude', type="args",
                   help='source files to be excluded from codestyle')
 
 
 def pytest_collect_file(parent, path):
     config = parent.config
-    if config.getoption('codestyle') and path.ext == '.py' and path not in config.getini('codestyle_exclude'):
-        return Item(path, parent)
+    if config.getoption('codestyle') and path.ext == '.py':
+        if not any(path.fnmatch(glob) for glob in config.getini('codestyle_exclude')):
```
Comment (henry0312) [inline pytest_codestyle.py]:
```suggestion
        if not any(path.fnmatch(pattern) for pattern in config.getini('codestyle_exclude')):
```

cf. https://py.readthedocs.io/en/latest/path.html?highlight=fnmatch#py._path.local.LocalPath.fnmatch

**Comment by TLATER** [comment]
Thanks for the review @henry0312 - sorry it took me so long to respond as well ;)

I've simply applied the changes you suggested, is that enough for a merge in your eyes?

**Comment by lockwooddev** [comment]
Will there be a new release for this change?

**Comment by henry0312** [comment]
I want to release a new version, but I've waited for #47.

**Pull request merged**